app/views/dashboards/analytics.erb

```diff
@@ -2,6 +2,34 @@
 
 <main id="main-content">
   <div class="dashboard-container analytics-container crayons-layout" id="user-dashboard">
+    <% if @organizations.any? %>
+      <nav aria-label="Organizations menu">
```

Comment:
```suggestion
      <nav aria-label="Dashboards">
```
suggestion (not blocking) - Looking at this as a user, I feel like "dashboards" better describes what this navigation set is - a list of dashboard links. It's not a super strong opinion though.